src/App/Hovedside/BeOmTilgang/SyfoBeOmTilgangModalBoks/SyfoBeOmTilgangModalBoks.less

```diff
@@ -17,8 +15,8 @@
         outline: none;
         color: white;
         text-decoration: none;
-        background-color: @fokusFarge;
-        box-shadow: 0 0 0 2px @fokusFarge;
+        background-color: var(--navds-global-color-blue-800);
```

Comment:
Her kunne man brukt `--navds-semantic-color-focus` for å beholde semantisk betydning

Reply:
de anbefaler at vi bruker de globale variabelene enn så lenge: 

> Semantiske farger beskriver hvordan fargene bør brukes. Alle semantiske farger er tilgjengelige for brukere av @navikt/ds-css-pakken som css custom properties. Disse er da prefikset, så brukes slik: var(--navds-semantic-color-interaction-primary). Vi blir å oppdatere hvordan vi hånterer tokens om ikke så alt for lenge, så utviklere anbefales å bruke globale css verdier for nå.